src/components/NumericalFacets.tsx

```diff
@@ -0,0 +1,105 @@
+import { FacetsProvider, RangeInput, RangeInputCssClasses } from './Filters';
+import { FilterGroup, FilterGroupCssClasses } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment, useMemo } from 'react';
+import { DisplayableFacet, NumberRangeValue } from '@yext/answers-headless-react';
+import { StandardFacetsProps } from './StandardFacets';
+import { isNumberRangeFilter } from '../models/NumberRangeFilter';
+
+/**
+ * The CSS class interface for {@link NumericalFacets}.
+ *
+ * @public
+ */
+export interface NumericalFacetsCssClasses extends FilterGroupCssClasses, RangeInputCssClasses {
+  container?: string,
+  divider?: string,
+  rangeInputContainer?: string,
+  rangeInputDivider?: string
+}
+
+/**
+ * Props for the {@link NumericalFacets} component.
+ *
+ * @public
+ */
+export interface NumericalFacetsProps extends StandardFacetsProps {
+  /**
+   * Returns the filter's display name based on the range values which is used when the filter
+   * is displayed by other components such as AppliedFilters.
+   *
+   * @remarks
+   * By default, the displayName separates the range with a dash such as '10 - 20'.
+   * If the range is unbounded, it will display as 'Up to 20' or 'Over 10'.
+   */
+  getFilterDisplayName?: (value: NumberRangeValue) => string,
+  /**
+   * An optional element which renders in front of the input text.
+   */
+  inputPrefix?: JSX.Element,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: NumericalFacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays numerical facets applicable to the current vertical search.
+ *
+ * @param props - {@link NumericalFacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function NumericalFacets({
+  searchOnChange,
+  excludeFieldIds = [],
```

Comment:
would we want this to be `includedFIeldIds` instead? I imagine most facets will not be numerical (or hierarchical) and it could be annoying for people to type out all but one fieldId that they have facets for. 

I feel like product may end up just asking for both excluded and included props, but maybe for now we can just have `includedFieldIds` for both hierarchical and numerical facets?

Reply:
numerical facets will only display filters with a value shape of `{start: blah, end: blah}` so it wouldn't overlap with Hierarchical/StandardFacets ones. `excludeFieldIds` is inherited as the prop extends `StandardFacetsProps`. hierarchicalFacets already have `fieldIds` (I can rename) since there's no good way to distinguish it from the normal ones without assuming with delimiters. I can see why it would be good to have for numerical ones too, I can add in `includedFieldIds` for numerical facets